src/app/domUtils.ts

```diff
@@ -113,14 +113,7 @@ export const scrollIntoView = (elem: HTMLElement) => {
 };
 
 export const elementDescendantOf = (element: Element, ancestor: Element): boolean => {
-  if (element === ancestor) {
-    return true;
-  }
-  if (!element.parentNode || !(element.parentNode instanceof assertWindow().Element)) {
-    return false;
-  }
-
-  return elementDescendantOf(element.parentNode, ancestor);
+  return ancestor.contains(element);
```

Comment:
should probably just refactor out the whole `elementDescendantOf` method and replace with this

Reply:
also out of curiosity what case was broken?

Reply:
`DisplayNote` is using `useOnClickOutside(element, isFocused, onBlur);` to blur when user clicks outside. This hook is using `elementDescendantOf` which is iterating over parents for clicked element and is comparing them to the passed element.
It will not work for elements which are rendered conditionally because at some point their `parentElement` becomes null when they are deattached from the DOM.

There is also another version of this pr which uses CSS to hide `Confirmation` components, but Malar said it was breaking some automation tests: https://github.com/openstax/rex-web/pull/640